network-config.yaml

```diff
@@ -1,48 +1,332 @@
+  - name: Boba
+    networks:
+      - name: Testnet
+        chainid: 28
+        id: boba-test
+        host: https://rinkeby.boba.network
+        explorer: https://blockexplorer.rinkeby.boba.network/api
+        multicall2: "0xeD188A73c442Df375b19b7b8f394a15a2b851BB5"
+      - name: Mainnet
+        chainid: 288
+        id: boba-main
+        host: https://mainnet.boba.network
+        explorer: https://blockexplorer.boba.network/api
+        multicall2: "0xbe2Be647F8aC42808E67431B4E1D6c19796bF586"
+  - name: Fantom Opera
+    networks:
+      - name: Testnet
+        chainid: 4002 
+        id: fantom-test
+        host: https://rpc.testnet.fantom.network
+        explorer: https://explorer.testnet.fantom.network/api
+      - name: Mainnet
+        chainid: 250
+        id: ftm-main
+        host: https://rpc.ftm.tools
+        explorer: https://api.ftmscan.com/api
+  - name: Harmony
+    networks:
+      - name: Mainnet (Shard 0)
+        chainid: 1666600000
+        host: https://api.harmony.one
+        id: harmony-main
+        multicall2: "0x3E01dD8a5E1fb3481F0F589056b428Fc308AF0Fb"
+  - name: Moonbeam
+    networks:
+      - name: Mainnet
+        chainid: 1284
+        id: moonbeam-main
+        host: https://moonbeam.api.onfinality.io/public
+        explorer: https://api-moonbeam.moonscan.io/api
+        multicall2: "0x1337BedC9D22ecbe766dF105c9623922A27963EC"
+      - name: Moonbase Alpha
+        chainid: 1287 
+        id: moonbeam-test
+        host: https://moonbeam-alpha.api.onfinality.io/public
+        explorer: https://api-moonbase.moonscan.io/api
+        multicall2: "0x37084d0158C68128d6Bc3E5db537Be996f7B6979"
+  - name: Moonriver
+    networks:
+      - name: Mainnet
+        chainid: 1285
+        id: moonriver-main
+        host: https://moonriver.api.onfinality.io/public
+        explorer: https://api-moonriver.moonscan.io/api
+        multicall2: "0xaef00a0cf402d9dedd54092d9ca179be6f9e5ce3"
+  - name: Optimistic Ethereum
+    networks:
+      - name: Mainnet
+        chainid: 10
+        id: optimism-main
+        host: https://mainnet.optimism.io
+        explorer: https://api-optimistic.etherscan.io/api
+        multicall2: "0x2DC0E2aa608532Da689e89e237dF582B783E552C"
+      - name: Kovan
+        chainid: 69
+        id: optimism-test
+        host: https://kovan.optimism.io
+        explorer: https://api-kovan-optimistic.etherscan.io/api
+        multicall2: "0x2DC0E2aa608532Da689e89e237dF582B783E552C"
+  - name: Polygon
+    networks:
+      - name: Mainnet (Infura)
+        chainid: 137
+        id: polygon-main
+        host: https://polygon-mainnet.infura.io/v3/68be3027359a4525a4414220492d931c
+        explorer: https://api.polygonscan.com/api
+        multicall2: "0xc8E51042792d7405184DfCa245F2d27B94D013b6"
+      - name: Mumbai Testnet (Infura)
+        chainid: 80001
+        id: polygon-test
+        host: https://polygon-mumbai.infura.io/v3/...... # < update here your projectID from infura
```

Comment:
https://polygon-mumbai.infura.io/v3/$WEB3_INFURA_PROJECT_ID 

then we can use an environment variable for that